tests/cases/fourslash/jsxTagLinkedEdit7.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path='fourslash.ts' />
+
+// @FileName: /fragment.tsx
+////const jsx = (
+////    /*7*/</*0*/>/*1*/
+////        <img />
+////    /*8*/</*2*///*3*/>/*4*/
+////);
+////const jsx2 = (
+////    <>/*5*/
+////    </>
+////);
+////const jsx3 = (
+////    </*6*/>
+////        <div>
+////    </>
+////);
+
+const linkedCursors7 = {ranges: [{start: 14, end: 14}, 

```

Comment:
The data here is currently incorrect because of the `pos + 1` bug I mentioned earlier. You need to make sure you get the start, not-including trivia (i.e. whitespace + comments).